OPHD/Population/Population.cpp

```diff
@@ -98,156 +99,146 @@ int Population::adults() const
 void Population::spawn_children(int morale, int residences, int nurseries)
 {
 	if (residences < 1 && nurseries < 1) { return; }
+	if (mPopulation[PersonRole::ROLE_SCIENTIST] + mPopulation[PersonRole::ROLE_WORKER] < 1) { return; }
 
-	// This should be adjusted to maybe two or three kids per couple to allow for higher growth rates.
-	if (mPopulation[PersonRole::ROLE_SCIENTIST] + mPopulation[PersonRole::ROLE_WORKER] > mPopulation[PersonRole::ROLE_CHILD])
-	{
-		mPopulationGrowth[PersonRole::ROLE_CHILD] += mPopulation[PersonRole::ROLE_SCIENTIST] / 4 + mPopulation[PersonRole::ROLE_WORKER] / 2;
+	mPopulationGrowth[PersonRole::ROLE_CHILD] += mPopulation[PersonRole::ROLE_SCIENTIST] / 4 + mPopulation[PersonRole::ROLE_WORKER] / 2;
 
-		int divisor = moraleModifierTable[moraleIndex(morale)].fertilityRate;
+	int divisor = moraleModifierTable[moraleIndex(morale)].fertilityRate;
```

Comment:
Curious, if values from that table are always used as divisors, why are any of the entries 0? Would it make more sense to set a minimum value of 1? If all values were non-zero, you might not need to guard expressions with `if` statements.